client/src/components/Login.tsx

```diff
@@ -58,11 +59,11 @@ export default function Login(): React.ReactElement {
 }
 
 const styles: Styles = {
-  loginContainer: {
+  loginContainer: css({
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
-  },
-  loginHeader: {},
-  loginText: {},
+  }),
+  loginHeader: '',
+  loginText: '',
```

Comment:
let's just remove these from the code for now since they aren't being used anywhere